### PR TITLE
[nearlib] add tests for loadcontract + bump verison

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

**Comment by janedegtiareva** [comment]
Test passes locally. There is some other problem which is causing test failures (on master too)

File: nearlib/test/integration.test.js [hunk 1]
```diff
@@ -117,11 +117,29 @@ describe('with deployed contract', () => {
...
         await nearjs.waitForTransactionResult(scheduleResult);
-        const secondViewFunctionResult = await nearjs.callViewFunction(
+        const getValueResult = await nearjs.callViewFunction(
             contractName,
             'getValue', // this is the function defined in hello.wasm file that we are calling
             {});
-        expect(secondViewFunctionResult).toEqual(setCallValue);
+        expect(getValueResult).toEqual(setCallValue);
+
+        // test that load contract works and we can make calls afterwards
```
Comment (vgrichina) [inline nearlib/test/integration.test.js]: why not make a separate test case?

Comment (janedegtiareva) [reply nearlib/test/integration.test.js]: I don't have a strong opinion either way. It's roughly the same amount of code

Comment (vgrichina) [reply nearlib/test/integration.test.js]: See e.g. `'can get logs from method result'` test

**Pull request merged**